upup/pkg/fi/cloudup/openstacktasks/instance.go

```diff
@@ -176,3 +190,50 @@ func (_ *Instance) RenderOpenstack(t *openstack.OpenstackAPITarget, a, e, change
 	klog.V(2).Infof("Openstack task Instance::RenderOpenstack did nothing")
 	return nil
 }
+
+func includeBootVolumeOptions(t *openstack.OpenstackAPITarget, e *Instance, opts servers.CreateOptsBuilder) (servers.CreateOptsBuilder, error) {
+	if bootFromVolume(e.Metadata) {
```

Comment:
Isn't it more idiomatic to short circuit and return earlier than wrap this whole thing in an if block?